src/main.rs

```diff
@@ -199,6 +199,27 @@ async fn main() -> Result<()> {
                     )
                     .await?
             }
+            #[cfg(feature = "capability-did-write")]
+            ("did_create", Some(sub_m)) => {
+                get_vade_evan(sub_m)?
+                    .helper_did_create(
+                        sub_m.value_of("bbs_key"),
+                        sub_m.value_of("signign_key"),
+                        sub_m.value_of("service_endpoint"),
```

Comment:
We also have a small helper that keeps the syntax a bit more aligned for optional arguments: `get_optional_argument_value`. ^^

Reply:
done